npm/README.md

```diff
@@ -0,0 +1,48 @@
+# npm
```

Comment:
Any way we can link this to: https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md#cypress-and-packages and/or update the contributing guide and link to that section here?

I'd prefer keep this doc & link in contributing doc

Reply:
Done, deleted most of this and linked there.